Loop/Managers/TestingScenariosManager.swift

```diff
@@ -242,7 +242,9 @@ extension TestingScenariosManager {
         if instance.hasCGMData {
             if let cgmManager = deviceManager.cgmManager as? TestingCGMManager {
                 if instance.shouldReloadManager?.cgm == true {
-                    testingCGMManager = reloadCGMManager(withIdentifier: cgmManager.pluginIdentifier)
+                    Task {
```

Comment:
No concern here with the wipeExistingData and subsequent calls from completing before this? It's definitely an edge case but could still happen. Could we scope the Task around a larger block?